python/lsst/ip/diffim/imagePsfMatch.py

```diff
@@ -416,14 +408,22 @@ def matchExposures(self, templateExposure, scienceExposure,
             if not templateExposure.hasPsf():
                 self.log.warning("No estimate of Psf FWHM for template image")
             else:
-                templateFwhmPix = self.getFwhmPix(templateExposure.getPsf())
+                templateFwhmPix = diffimUtils.evaluateMeanPsfFwhm(
+                    templateExposure,
+                    fwhmExposureBuffer=self.kConfig.fwhmExposureBuffer,
+                    fwhmExposureGrid=self.kConfig.fwhmExposureGrid
+                )
                 self.log.info("templateFwhmPix: %s", templateFwhmPix)
 
         if scienceFwhmPix is None:
             if not scienceExposure.hasPsf():
                 self.log.warning("No estimate of Psf FWHM for science image")
             else:
-                scienceFwhmPix = self.getFwhmPix(scienceExposure.getPsf())
+                scienceFwhmPix = diffimUtils.evaluateMeanPsfFwhm(
+                    scienceExposure,
+                    fwhmExposureBuffer=self.kConfig.fwhmExposureBuffer,
+                    fwhmExposureGrid=self.kConfig.fwhmExposureGrid
+                )
```

Comment:
Should these calls to `evaluateMeanPsfFwhm` also be changed then? And if so, should I keep the `getFwhmPix` that I removed in the PR or switch to using `getPsfFwhm`? It looks like there are two ways of doing very similar things and I don't know if that's intentional.